lib/coverband/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module Coverband
-  VERSION = '3.0.1'
+  VERSION = '3.0.2.alpha'
```

Comment:
Are we targeting this for 3.0.2 or 4.x?

Reply:
I wasn't sure yet but needed to pick something to test locally, I didn't release the alpha gem. I think to really get full benefits we will want to do some breaking changes so I am going to move it to 4.0.0

Reply:
moved to 4.0.x in latest commits